recipes/elk/package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "id": "elk",
+  "name": "Elk",
+  "version": "1.0.1",
```

Comment:
you can override that check by committing the change with the `-n` switch for `git commit` command